mantidproject/mantid
# Add GSAS tab to Engineering GUI documentation

Sections on the new GSAS tab were added to the docs for the ED GUI

**To test:**

Build the docs and make sure they look good. Have a read and make sure everything in there sounds sensible/lucid

Fixes #22044 

**Release Notes** 
Doesn't need to be in the release notes, as Engineering scientists aren't using the GUI yet anyway

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
